snippet/getURLParameters.md

```diff
@@ -0,0 +1,24 @@
+---
+title: getURLParameters
+tags: Dart,string,uri,intermediate
+---
+Returns an String containing the parameters of the current URL or an empty String in case tehe parameter doesn´t contain parameters or is not a proper URI.
+
+Use `String` concatanation and `Map<String, String> get queryParameters` to decode key and value from de url.
+
+```dart
+String getURLParameters (String url) {
```

Comment:
I believe we should return a `Map` just like we do in JS, so that the results are usable. Returning a `String` is not ver useful.

Reply:
Yes, I agree.

Reply:
Now I would like to accept and "Commit suggestion" but I did not found this option. 
As I am new in open source project like this one, may be I am not doing it correctly. Anyway, I Appreciate your help.

Reply:
Could you please help me with this. I have already corrected this as you have just suggested.I wonder how can I update this. Thanks.